pages/learn/index.vue

```diff
@@ -24,27 +24,35 @@
       @top-filter-changed="setLearnLevel"
       @aside-filter-changed="setTimeScale"
     >
-      <TheCarefulExplanationForBeginners
-        v-if="isShowingOneMinuteFor(learnLevels.beginner) && !isShowingEverything"
-        :compact="isShowingMoreResources"
-        url="/learn/?learnLevel=Beginner&amp;timeScale=1%20minute"
-      />
-      <TheCarefulExplanationForExperts
-        v-if="isShowingOneMinuteFor(learnLevels.advanced) && !isShowingEverything"
-        :compact="isShowingMoreResources"
-        url="/learn/?learnLevel=Advanced&amp;timeScale=1%20minute"
-      />
-      <AppCard
-        v-for="resource in filteredLearningResources"
-        :key="resource.path"
-        :image="resource.image"
-        :title="resource.title"
-        :to="resource.to"
-        :cta-label="resource.ctaLabel"
-        class="app-filters-results-layout__results-item"
-      >
-        <nuxt-content :document="resource" />
-      </AppCard>
+      <div class="app-filters-results-layout__careful-explanation">
+        <TheCarefulExplanationForBeginners
+          v-if="isShowingOneMinuteFor(learnLevels.beginner) && !isShowingEverything"
+          :compact="isShowingMoreResources"
+          url="/learn/?learnLevel=Beginner&amp;timeScale=1%20minute"
+        />
+        <TheCarefulExplanationForExperts
+          v-if="isShowingOneMinuteFor(learnLevels.advanced) && !isShowingEverything"
+          :compact="isShowingMoreResources"
+          url="/learn/?learnLevel=Advanced&amp;timeScale=1%20minute"
+        />
+      </div>
+      <div class="bx--row">
+        <div
+          v-for="resource in filteredLearningResources"
+          :key="resource.path"
+          class="bx--col-sm-4 bx--col-xlg-8"
+        >
+          <AppCard
+            :image="resource.image"
+            :title="resource.title"
+            :to="resource.to"
+            :cta-label="resource.ctaLabel"
+            class="app-filters-results-layout__results-item app-filters-results-layout__results-item_2-cols-xlg"
```

Comment:
Why did you add this `app-filters-results-layout__results-item_2-cols-xlg`? It's not enough with adding

```
@include mq($from: x-large) {
     height: calc(100% - #{$layout-02});
 }
```

to the `&__results-item`?

Reply:
Because `app-filters-results-layout__results-item` is also used in the **Events Page**, which currently doesn't have the "max" layout, so it would look like this:

![Screenshot 2021-05-25 at 12 57 11](https://user-images.githubusercontent.com/22047320/119486609-ba948780-bd58-11eb-98ca-f8bea7802bed.png)

Reply:
great! thank you!